xfer/collector.go

```diff
@@ -17,51 +17,61 @@ const (
 
 var (
 	// MaxBackoff is the maximum time between connect retries.
-	MaxBackoff = 2 * time.Minute // externally configurable.
+	// It's exported so it's externally configurable.
+	MaxBackoff = 2 * time.Minute
+
+	tick = time.Tick
```

Comment:
why have this? It's used exactly once, and makes the call site just that little bit more obscure.